### ISSUE TITLE
Incorrect Indentation for if statements with else

### ISSUE THREAD

**Issue by grappler** [squizlabs/PHP_CodeSniffer#1576]
I am getting the following error for `if ( is_customize_preview() ) :`

```
11 | ERROR | [x] Line indented incorrectly; expected 0 tabs, found 1
    |       |     (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
```

When I remove `<?php else : ?>` the error goes away.

```php
<div>
	<?php
	if ( is_front_page() ) :
	?>

	<?php else : ?>

	<?php
	endif;

	if ( is_customize_preview() ) :
	?>
	<?php endif; ?>
</div><!-- .site-branding -->
```

I have tested this with `2.9.2` and `3.0.2`.

**Comment by gsherwood** [comment]
The mix of indents from scope blocks and closing and opening PHP tags is throwing the sniff completely off and I'm not currently sure if it is expected or not (I have not had time to look into this yet).

Either way, this is actually how that sniff wants you to write your code:
```php
<div>
    <?php
    if ( is_front_page() ) :
        ?>

        <?php
    else :
        ?>

        <?php
    endif;

    if ( is_customize_preview() ) :
        ?>
        <?php
    endif; 
    ?>
</div><!-- .site-branding -->
```
But it doesn't enforce the indent of the open/close tags because people tend to want to put them in places that line up with the HTML/XML/other code inside the code blocks. For this reason, sniffs like ScopeIndent are rarely successful in template files unless you treat your open and close tags in the same way as the sample code above.

**Comment by grappler** [comment]
Thank you! 😄
